bucket/innounp.json

```diff
@@ -1,8 +1,8 @@
 {
     "homepage": "http://innounp.sourceforge.net",
-    "version": "0.46",
-    "url": "https://r15ch13.keybase.pub/scoop/innounp/innounp046.rar",
-    "hash": "3c056e2fc46ca9e92405f13ddf1f250cfe4bd66cb83b6647d87c74bc7242b121",
+    "version": "0.47",
+    "url": "https://sourceforge.net/projects/innounp/files/innounp/innounp%200.47/innounp047.rar",
```

Comment:
This should be https://downloads.sourceforge.net/project/innounp/files/innounp/innounp%200.47/innounp047.rar per https://github.com/lukesampson/scoop/pull/2168/commits/20c0faa57c16ec3ac04ddfa7df2d6eafa1dbb99e